lib/actionpack/all/actionpack.rbi

```diff
@@ -132,11 +132,56 @@ class ActionController::Parameters
   sig { params(block: T.untyped).returns(T.untyped) }
   def reject(&block); end
 
-  sig { params(key: T.any(Symbol, T::Array[Symbol])).returns(ActionController::Parameters) }
+  sig do
```

Comment:
This is more accurate. Returning `ActionController::Parameters` is outright wrong and can make people distrust the type system.